StripeIdentity/StripeIdentity/Source/NativeComponents/ViewControllers/DocumentCaptureViewController.swift

```diff
@@ -53,51 +53,77 @@ final class DocumentCaptureViewController: IdentityFlowViewController {
 
     // MARK: Computed Properties
 
+    private var lastScanningInstructionText: String?
+    private var lastScanningInstructionTextUpdate = Date.distantPast
+
+    private func resetLastScanningInstructionText() {
+        lastScanningInstructionText = nil
+        lastScanningInstructionTextUpdate = Date.distantPast
+    }
+
     var viewModel: DocumentCaptureView.ViewModel {
         switch imageScanningSession.state {
         case .initial:
+            resetLastScanningInstructionText()
             return .scan(
                 .init(
                     scanningViewModel: .blank,
                     instructionalText: scanningInstructionText(
                         for: .front,
-                        idDetectorOutput: nil
+                        documentScannerOutput: nil
                     )
                 )
             )
-        case .scanning(let documentSide, let idDetectorOutput):
+        case .scanning(let documentSide, let documentScannerOutput):
+            let newScanningInstructionText: String
+            let now = Date()
+            if  now.timeIntervalSince(lastScanningInstructionTextUpdate) > 1 {
```

Comment:
nit: there are two spaces here, i'm surprised this didn't get caught by our linter?

also, would be helpful here to put a comment like: // update instruction text, at most once a second